style.css

```diff
@@ -94,6 +94,28 @@
             padding: 1rem;
         }
 
+        #civselect {
+            font-size: 24px;
+            font-weight: bold;
+            font-family: 'Times New Roman', Times, serif;
+            background-color: #72604a;
+            flex: 1 0 auto;
+            margin-right: 1rem;
+            color: white;
+            text-shadow: #000000 -3px 2px;
+            box-shadow: inset 0 2px #3f4806, inset 2px -2px #768141, inset -2px 0 #768141;
+            padding: 5px;
+        }
+
+        #civselect option {
+            color: black;
+            background-color: white;
+        }
```

Comment:
With some browsers I am getting black text with a black shadow, which is illegible.

I actually prefer the white on brown text, so I would be in favour of removing this part entirely.

Reply:
Done.